servicetalk-http-api/src/main/java/io/servicetalk/http/api/StreamingHttpPayloadHolder.java

```diff
@@ -133,6 +134,61 @@ void transformMessageBody(UnaryOperator<Publisher<?>> transformer) {
         messageBody = transformer.apply(messageBody());
     }
 
+    <T, S> void transform(final TrailersTransformer<T, S> trailersTransformer,
+                          final HttpStreamingDeserializer<S> serializer) {
+        if (messageBody == null) {
+            messageBody = defer(() ->
+                    from(trailersTransformer.payloadComplete(trailersTransformer.newState(),
+                            headersFactory.newEmptyTrailers())).subscribeShareContext());
+        } else {
+            final Publisher<?> oldMessageBody = messageBody;
+            messageBody = defer(() -> {
```

Comment:
1. The `payloadInfo.setEmpty(false)` is missed compare to the `transform(TrailersTransformer)` overload.
2. Is there any way to share code between both `transform` overloads?